generators/app/index.js

```diff
@@ -197,6 +197,11 @@ module.exports = class extends BaseBlueprintGenerator {
             type: Boolean,
         });
 
+        this.option('spring-boot23', {
+            desc: 'Use spring boot 2.3 (beta)',
```

Comment:
we should not rely on specific branch here... otherwise, next versions, we'll add spring boot 2.4, 2.5, 2.6 etc etc

Reply:
Will be removed before merging the PR, and added if a partial spring-boot 2.3 support is added.

Reply:
Removed